code/modules/clothing/under/jobs/security.dm

```diff
@@ -199,3 +199,20 @@
 	icon_state = "pod_pilot"
 	item_state = "pod_pilot"
 	item_color = "pod_pilot"
+
+//Drill Instructor
+/obj/item/clothing/under/rank/security/instructor
+	name = "drill instructor's jumpsuit"
+	desc = "A buttoned red shirt tucked into black slacks, it bears the Security Patch on it's sleeves"

```

Comment:
`its` not `it's`.

Reply:
This was copypasted from the person that asked for it